ipn/localapi/localapi.go

```diff
@@ -2208,10 +2208,20 @@ func (h *Handler) serveWeb(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, err.Error(), http.StatusInternalServerError)
 			return
 		}
+		// try to set pref, but ignore errors
+		_, _ = h.b.EditPrefs(&ipn.MaskedPrefs{
+			Prefs:           ipn.Prefs{RunWebClient: true},
+			RunWebClientSet: true,
+		})
```

Comment:
oh nice! I was wondering about this - I guess the alternative is we just let them set it in the prefs endpoint

Reply:
yeah, I figured we'll eventually expose it as a flag to `tailscale set` as well.  But I think setting the pref here as well makes sense, since you've explicitly started the web client at this point.